newscorp-ghfb/NewsKit
# Feat/ppdsc 1939 Make <title &  meta description statically rendered

PPDSC-1939

**What**

Fixed bug where the meta tags wrapped in <Head were rendered client side rather than statically.
  used getStaticProps in _document.
  
 Created New component HeadNextSeo for refactoring the usage of meta tags across the repo and implemented NextSeo for easily populating meta tags in future.





**I have done:**
 - [ ] Written accessibility tests against the component
 - [ ] Updated the release notes for the [next release version](https://nidigitalsolutions.jira.com/wiki/spaces/NPP/pages/2349695021/Release%2BNotes%2BCode%2B-%2BWeb%2BDocumentation)
 - [ ] Discussed with QA and/or design about any visual changes

**I have tested manually:**
 - [ ] Renders correctly on IE11
 - [ ] The feature's functionality is working as expected on other browsers such as Safari, Firefox and IE11
 - [ ] There are no accessibility errors reported by WAVE
 - [ ] The screen reader reads the expected text for the tabbable elements.





**Before**


**After:**


**Who should review this PR**


**How to test**

Run `yarn build:doce` , in the new public folder created you should be finding only the specific meta tags for every page
